src/dao/migrations/20170909162220_create-installation-table.js

```diff
@@ -0,0 +1,22 @@
+const logger = require('~/src/logging').logger(module)
+const Installation = require('~/src/models/github/Installation')
+
+const INSTALLATION_TABLE_NAME = Installation.typeName
+
+exports.up = function (knex, Promise) {
+  logger.info('Attempting to run "up" on "create-installation-table"')
```

Comment:
No -I followed the existing pattern here: https://github.com/windbreaker-io/windbreaker-hooks/blob/master/src/dao/migrations/20170714072908_create-webhook-table.js I left it in intentionally for debugging purposes (only runs on startup).

Reply:
Gotcha, it seemed like a pretty similar message to the one below.

Reply:
Ah, now I see that you're pointing out the fact that there are two log statements, rather than suggesting to remove them entirely. These could be combined, I suppose. A future PR could combine them, but I was just being consistent.